test/functional/bundleadd/add-alias-basic.bats

```diff
@@ -0,0 +1,310 @@
+#!/usr/bin/env bats
+
+# Author: <author name>
```

Comment:
it would be nice if you add your contact information here in case someone has a question regarding these tests in the future

Reply:
Ah thanks, I missed that.